### DISCUSSION TITLE
Disable X-Ratelimit-Limit: change the limit or even disable the rate limit of 50 requests per minute

### DISCUSSION THREAD

**Discussion by kyrill-bo** [sinamics/ztnet#550] [General]
Hi there, 

I would like to deactivate the rate limit of 50 requests per minute for the api key. 
I'm currently using the docker ztnet.

How do I do this?

**Comment by sinamics** [comment]
That is not possible at the moment.
Create a new feature request, then this might be added.

**Reply by kyrill-bo** [reply]
i wouldn't need it if i could find out which managed ip a network_member has been assigned. as you can read ipAssignments from https://ztnet.network/Rest%20Api/Personal/Network-Members/get-network-member-info. 

Where can I find it in postgres, if it is stored there?

**Reply by sinamics** [reply]
node`s managed ip are not stored in postgres, its retrived from the controller. You can see all params that is preserved in db here:https://github.com/sinamics/ztnet/blob/5fd0beef26602c2adcc4a791c69e28ac580af354/prisma/schema.prisma#L78-L93
